### PR TITLE
GTFS+ Editor - Route Attribute docs

### PR DESCRIPTION
**Pull request by binh-dam-ibigroup**
### Checklist

- [x] Appropriate branch selected _(This doc PR is against `mtc-docs`)_
- [na] Any modified or new methods or classes have helpful JSDoc and code is thoroughly commented
- [na] The description lists all applicable issues this PR seeks to resolve
- [na] The description lists any configuration setting(s) that differ from the default settings
- [na] All tests and CI builds passing

### Description

This PR adds a section about GTFS+ route attribute editing to the GTFS+ editor documentation per client request.

### CODE REVIEW

File: docs/user/gtfs-plus-editor.md [hunk 1]
```diff
@@ -33,3 +33,69 @@ After making changes to the data, the "Save and Revalidate" button can be used t
...
+
+
+
+
+
+
+
+
+
+
```
Comment (miles-grant-ibigroup) [inline docs/user/gtfs-plus-editor.md:101]: We can probably remove these blank lines right?

**Review by miles-grant-ibigroup — APPROVED** [review]
Very thorough documentation! Looks great

File: docs/user/gtfs-plus-editor.md (hunk 1 above)
Comment (binh-dam-ibigroup) [reply docs/user/gtfs-plus-editor.md]: Yes, thank you! (https://github.com/ibi-group/datatools-ui/pull/796/commits/7a5b8f319da46d48a0f479e42740fecf578168f1)

File: docs/user/gtfs-plus-editor.md [hunk 2]
```diff
@@ -33,3 +33,59 @@ After making changes to the data, the "Save and Revalidate" button can be used t
...
+From the Feed Version viewer, clicking the "GTFS+ for this version" shortcut on the left side of the screen will open the GTFS+ summary page that lists GTFS+ files and validation summary.
+
+If editing `route_attributes.txt` from scratch, the row for `route_attributes.txt` will be grayed out in the GTFS+ summary page. Click "Edit GTFS+" to the top-right corner to open the GTFS+ Editor.
+
+Note that route attributes can only be added for the transit routes defined for the active agency feed. To edit route attributes for another agency, navigate back to the applicable project and select the desired agency and feed version.
+
+### Route Attributes Editor Overview
+
+Once in the GTFS+ editor, select "route_attributes" from the GTFS+ files shown on the left side. "route attributes" will be grayed-out but still available for editing if no data were previously entered.
```
Comment (daniel-heppner-ibigroup) [inline docs/user/gtfs-plus-editor.md:49]: Add underscore to `route attributes`. I think.

**Review by daniel-heppner-ibigroup — APPROVED** [review]
Easy to read and made perfect sense to me!

File: docs/user/gtfs-plus-editor.md (hunk 2 above)
Comment (binh-dam-ibigroup) [reply docs/user/gtfs-plus-editor.md]: Good find, thanks! Fixed in https://github.com/ibi-group/datatools-ui/pull/796/commits/41e95f9eadc582017c801d7b6d0ad7ac5843aeff.

**Pull request merged**